packages/loot-design/src/components/NotesButton.js

```diff
@@ -20,22 +21,39 @@ export function NotesTooltip({
   let inputRef = React.createRef();
 
   useEffect(() => {
-    inputRef.current.focus();
-  }, []);
+    if (editable) {
+      inputRef.current.focus();
+    }
+  }, [inputRef, editable]);
 
   return (
     <Tooltip position={position} onClose={() => onClose(notes)}>
-      <textarea
-        ref={inputRef}
-        {...css({
-          border: '1px solid ' + colors.border,
-          minWidth: 300,
-          minHeight: 120,
-          outline: 'none'
-        })}
-        value={notes || ''}
-        onChange={e => setNotes(e.target.value)}
-      ></textarea>
+      {editable ? (
+        <textarea
+          ref={inputRef}
+          {...css({
+            border: '1px solid ' + colors.border,
+            padding: 8,
```

Comment:
Since there’s a 1px border, the padding should be decreased by 1 so the text remains aligned (or the padding of the `<Text>` should be 9)

```suggestion
            padding: 7,
```